source/backend/dal/Repositories/AcquisitionFileChecklistRepository.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Pims.Core.Extensions;
+using Pims.Dal.Entities;
+using Pims.Dal.Helpers.Extensions;
+
+namespace Pims.Dal.Repositories
+{
+    /// <summary>
+    /// Provides a repository to interact with acquisition file checklist items within the datasource.
+    /// </summary>
+    public class AcquisitionFileChecklistRepository : BaseRepository<PimsAcquisitionChecklistItem>, IAcquisitionFileChecklistRepository
+    {
+        #region Constructors
+
+        /// <summary>
+        /// Creates a new instance of a AcquisitionFileChecklistRepository, and initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="dbContext"></param>
+        /// <param name="user"></param>
+        /// <param name="logger"></param>
+        public AcquisitionFileChecklistRepository(PimsContext dbContext, ClaimsPrincipal user, ILogger<AcquisitionFileChecklistRepository> logger)
+            : base(dbContext, user, logger)
+        {
+        }
+
+        #endregion
+
+        #region Methods
+
+        public IEnumerable<PimsAcqChklstItemType> GetAllChecklistItemTypes()
+        {
+            return Context.PimsAcqChklstItemTypes
+                .Include(it => it.AcqChklstSectionTypeCodeNavigation)
+                .OrderBy(it => it.DisplayOrder)
+                .AsNoTracking()
+                .ToArray();
+        }
+
+        public List<PimsAcquisitionChecklistItem> GetChecklistItemsByAcquisitionFileId(long acquisitionFileId)
```

Comment:
nit: I think we are trying to use GetAll.... for any repo function that returns a list, so GetAllChecklistItemsByAcquisitionFileId

Reply:
will update